configure.ac

```diff
@@ -7,6 +7,10 @@ AC_INIT([hootenanny], [VERSION_NUMBER], [https://github.com/ngageoint/hootenanny
 AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([hoot-core/src])
 
+# use C++ 11
+AC_LANG([C++])
+AX_CXX_COMPILE_STDCXX_11([], [mandatory])
+
 ###
```

Comment:
When I was trying to use autotools to link to a different lib, libaddressinput (which I didn't end up using), I realized autotools wasn't compiling with c++ 11 causing the compilation of it to fail.  Even though the two new libs, libpostal and libphonenumber, don't need c++11, I thought it was a good idea to make autotools match how we compile hoot.

Reply:
+1

Reply:
Note that this m4 macro is not included in the default `autoconf` package, but `autoconf-archive`. This was added in 09515e7 and to our build containers/RPM requirements in ngageoint/hootenanny-rpms@7eecaef.